magefile.go

```diff
@@ -115,10 +115,25 @@ func Spec() error {
 func Integration(testName string) error {
 	switch testName {
 	case "steelthread":
-		return steelthread.RunTest()
-	default:
+		return integtest.RunSteelthreadTest()
+	case "didweb":
+		return integtest.RunDidWebTest()
+	case "all":
 		fmt.Println("Running all tests")
-		return steelthread.RunTest()
+		err := integtest.RunSteelthreadTest()
+		if err != nil {
+			fmt.Printf("Error running steelthread test")
+			return err
+		}
+		err = integtest.RunDidWebTest()
+		if err != nil {
+			fmt.Printf("Error running did:web steelthread test")
```

Comment:
I don't remember are we using logrus for the rest of this file? I think print is fine

Reply:
I use logrus everywhere now